src/test/kotlin/com/jetbrains/snakecharm/fixtures/SnakemakeResolveTestCase.kt

```diff
@@ -33,35 +24,17 @@ abstract class SnakemakeResolveTestCase : SnakemakeTestCase() {
             }
             return element as T
         }
-
-        protected fun assertIsBuiltin(element: PsiElement?) {
-            assertNotNull(element)
-            assertTrue(PyBuiltinCache.getInstance(element).isBuiltin(element))
-        }
-    }
-
-    protected abstract fun doResolve(): PsiElement?
-
-    protected fun <T : PsiElement> assertResolvesTo(langLevel: LanguageLevel,
-                                                    aClass: Class<T>,
-                                                    name: String): T {
-        val result = Ref<T>()
-
-        runWithLanguageLevel(
-                langLevel,
-                Runnable { result.set(assertResolvesTo(aClass, name, null)) }
-        )
-
-        return result.get()
     }
 
-    protected fun <T : PsiElement> assertResolvesTo(aClass: Class<T>, name: String): T {
-        return assertResolvesTo(aClass, name, null)
+    private fun doResolve(): PsiElement? {
+        val ref = fixture?.getReferenceAtCaretPosition("resolve/" +
+                getTestName(false) + ".smk")
+        return ref?.resolve()
     }
```

Comment:
Please
1) inline `ref` variable 
2) convert function to expression body
3) replace string concatenation with Kotlin style, e.g. `foo/$arg/boo/${foo(arg2)}/doo.smk`

Reply:
done